DockerfileTest

```diff
@@ -2,6 +2,8 @@ FROM maven:3-jdk-8
 
 RUN mkdir -p /usr/src/app/
 RUN mkdir -p /usr/src/run/
+RUN apt-get update && apt-get install
+RUN apt-get install dos2unix
```

Comment:
You shouldn't need this separate line.  You can move `dos2unix` to the above line.

Reply:
I only separated this for readability.

Reply:
Then you don't need the `apt-get install` on the first line.  Although, I believe the preferred method is to put the `apt-get update` and all installs on the same line.

Reply:
👍